lib/ansible/modules/windows/win_package.ps1

```diff
@@ -25,6 +25,7 @@ $validate_certs = Get-AnsibleParam -obj $params -name "validate_certs" -type "bo
 $creates_path = Get-AnsibleParam -obj $params -name "creates_path" -type "path"
 $creates_version = Get-AnsibleParam -obj $params -name "creates_version" -type "str"
 $creates_service = Get-AnsibleParam -obj $params -name "creates_service" -type "str"
+$isAllUsers = Get-AnsibleParam -obj $params -name "AllUsers" -type "bool" -default $true -aliases "isAllUser"
```

Comment:
When creating module options it's best to keep them align with the Ansible standard of snake_case, e.g. `install_all_users` or something like that. It is also best to have the variable reflect the same format of the actual option and keep it aligned to the existing standard in the module.